python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
```

Comment:
I don't feel so strongly about it, so happy to revert to @holdenk's suggested approach. It is more "boilerplate" to override each concrete class in the same way, but it does match the Scala side.